NYPLCardCreator/Classes/UserCredentialsViewController.swift

```diff
@@ -122,6 +124,15 @@ final class UserCredentialsViewController: TableViewController {
     self.activityView.isHidden = false
   }
   
+  func endSignIn() {
```

Comment:
This is a minor thing, but the ad hoc indenting here makes it a little hard for me to read this code. In particular, `completion:` is not lined up at all with the other keyword arguments and it gets lost. I might consider preferring one of these two styles instead:

```swift
func endSignIn() {
  UIView.transition(
    with: self.activityView,
    duration: 0.5,
    options: .transitionCrossDissolve,
    animations: {
      self.activityView.isHidden = true
    },
    completion: nil
  )
}
```

```swift
func endSignIn() {
  UIView.transition(
    with: self.activityView,
    duration: 0.5,
    options: .transitionCrossDissolve,
    animations: {
      self.activityView.isHidden = true
    },
    completion: nil)
}
```